src/js/containers/agency/AgencyContainerV2.jsx

```diff
@@ -187,22 +146,12 @@ export const AgencyProfileV2 = ({
                             <span>Fiscal Year</span>
                         </div>
                         <hr />
-                        <div className="sticky-header__toolbar-item">
-                            <Picker
-                                dropdownDirection="left"
-                                options={socialSharePickerOptions}
-                                selectedOption="copy"
-                                backgroundColor="#4A4A4A"
-                                sortFn={() => 1}>
-                                <FontAwesomeIcon icon="share-alt" size="lg" />
-                            </Picker>
-                            <span>Share</span>
-                            {showConfirmationText && (
-                                <span className="copy-confirmation">
-                                    <FontAwesomeIcon icon="check-circle" color="#3A8250" /> Copied!
-                                </span>
-                            )}
-                        </div>
+                        <ShareIcon
+                            slug={slug}
+                            email={{
+                                subject: `Check out Agency ${params.agencyId} on USAspending.gov!`,
```

Comment:
It would be nice to eventually change this to the agency name instead of its code. (Can be done later since the overview section is still in the backlog).